clients/src/main/resources/common/message/DeleteAclsRequest.json

```diff
@@ -16,7 +16,7 @@
 {
   "apiKey": 31,
   "type": "request",
-  "listeners": ["zkBroker"],
+  "listeners": ["zkBroker", "broker", "controller"],
```

Comment:
As mentioned, I am wondering if this should be `listeners": ["zkBroker", "broker"],` since the KRaft broker never forward DescribeAcls to the controller.

Reply:
(See above comment)